lib/macho/macho_file.rb

```diff
@@ -556,7 +556,7 @@ def low_fileoff
 
       segments.each do |seg|
         sections(seg).each do |sect|
-          next if sect.size == 0
+          next if sect.size.zero?
```

Comment:
Actually, it looks like RuboCop mistakenly thought that `sect` was an array/enumerable here - it's actually just a `Section{,64}` object, and so it doesn't have an `#empty?` method. I can define one, if you prefer that over `size.zero?`.